src/widgets/PaymentSummary.js

```diff
@@ -109,9 +109,14 @@ const PaymentSummaryComponent = ({
   usingInsurance,
   usingPaymentTypes,
 }) => {
-  const policyNumbers = React.useMemo(() => insurancePolicies.map(policy => policy.policyNumber), [
-    insurancePolicies,
-  ]);
+  const policyNumbers = React.useMemo(
```

Comment:
is there a reason for the change?  I like the terse version, so am just wondering if I'm missing something

Reply:
No reason. Was initially going to apply the policy number mapping here, but then moved it to the selector. 94c61daa9a484f4e517714b525636827e4c11b08.